CITATION.cff

```diff
@@ -0,0 +1,15 @@
+# This CITATION.cff file was generated with cffinit.
+# Visit https://bit.ly/cffinit to generate yours today!
+
+cff-version: 1.2.0
+title: SpinalHDL
+message: >-
+  If you use this software, please cite it using the
```

Comment:
This can be interpreted as a license : 
"If you use it, then you have to cite it"

What's about this wording instead : 
"If you want to cite this software, please use the metadata from this file."

Reply:
It was default-generated by the tool, I think you are very free to change it as you please. And I agree to that very much (and will adopt this to my repository too :joy: )

Reply:
I think the message thing is generated from the website, it makes the fields easily accessible. You should be able to freely fill it out as you like -- there is some 'validity' checked in the online tool (e.g., duplicate fields).